<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grip (A Graphic tool for image processing)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Main/Icons/Flower-power-green.svg</normaloff>:/Main/Icons/Flower-power-green.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_canvas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>78</width>
      <height>72</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>erase</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="lb_dummy">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar_main">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_new"/>
   <addaction name="act_load"/>
   <addaction name="act_save"/>
   <addaction name="act_exit"/>
   <addaction name="separator"/>
   <addaction name="act_copy"/>
   <addaction name="act_paste"/>
   <addaction name="act_clear"/>
   <addaction name="separator"/>
   <addaction name="act_play"/>
   <addaction name="act_stop"/>
   <addaction name="separator"/>
   <addaction name="act_settings"/>
   <addaction name="separator"/>
   <addaction name="act_help"/>
  </widget>
  <widget class="QDockWidget" name="dock_operators">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Operators</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tab_operators">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_custom">
        <attribute name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/3rd-party/Icons/Flower-power-green.svg</normaloff>:/3rd-party/Icons/Flower-power-green.svg</iconset>
        </attribute>
        <attribute name="title">
         <string>Custom</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolBox" name="tool_custom">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>190</width>
              <height>384</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Page 1</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_skimage">
        <attribute name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/3rd-party/Icons/skimage_logo.png</normaloff>:/3rd-party/Icons/skimage_logo.png</iconset>
        </attribute>
        <attribute name="title">
         <string>scikit-image</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolBox" name="tool_skimage">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>190</width>
              <height>384</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Page 1</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_opencv">
        <attribute name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/3rd-party/Icons/opencv_logo.png</normaloff>:/3rd-party/Icons/opencv_logo.png</iconset>
        </attribute>
        <attribute name="title">
         <string>OpenCV</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolBox" name="tool_opencv">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>190</width>
              <height>384</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Other</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="act_load">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/load.svg</normaloff>:/Main/Icons/load.svg</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load a processing file</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/exit.svg</normaloff>:/Main/Icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit program</string>
   </property>
  </action>
  <action name="act_play">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/play.svg</normaloff>:/Main/Icons/play.svg</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Execute pipe</string>
   </property>
  </action>
  <action name="act_new">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/new.svg</normaloff>:/Main/Icons/new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New processing</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a dataset</string>
   </property>
  </action>
  <action name="act_clear">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/clear.svg</normaloff>:/Main/Icons/clear.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear actual processing</string>
   </property>
  </action>
  <action name="act_help">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/help.svg</normaloff>:/Main/Icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="act_settings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/settings.svg</normaloff>:/Main/Icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Configure</string>
   </property>
  </action>
  <action name="act_stop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/pause.svg</normaloff>:/Main/Icons/pause.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="act_save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/save.svg</normaloff>:/Main/Icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the pipeline</string>
   </property>
  </action>
  <action name="act_copy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/copy.svg</normaloff>:/Main/Icons/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy selected items</string>
   </property>
  </action>
  <action name="act_paste">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Main/Icons/paste.svg</normaloff>:/Main/Icons/paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste clipboard items</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
